lib/account.rb

```diff
@@ -0,0 +1,27 @@
+class Account
+  attr_reader :entries
+  attr_reader :balance
+
+  def initialize(entries = [], balance = 2000)
+    @entries = entries
+    @balance = balance
+  end 
+
+  def add_entry(entry)
+    @entries << entry
+  end
+
+  def print_statement
```

Comment:
`print_statement` is a little hard to read because it is quite procedural.  Could it be made more declarative?